src/main/java/com/softserveinc/ita/utils/runners/TestRunner.java

```diff
@@ -1,23 +1,17 @@
 package com.softserveinc.ita.utils.runners;
 
 import com.codeborne.selenide.Configuration;
-import com.softserveinc.ita.pageobjects.CheckoutPage;
-import com.softserveinc.ita.pageobjects.HomePage;
-import com.softserveinc.ita.pageobjects.ProductPage;
 import org.testng.annotations.BeforeMethod;
 
 import static com.codeborne.selenide.Selenide.open;
+import static java.time.Duration.ofSeconds;
 
 public abstract class TestRunner {
 
-    protected HomePage homePage = new HomePage();
-    protected ProductPage productPage = new ProductPage();
-    protected CheckoutPage checkoutPage = new CheckoutPage();
-
     @BeforeMethod
     public void openHomePage() {
         Configuration.browserSize = "1920x1080";
-        Configuration.timeout = 8000;
+        Configuration.timeout = ofSeconds(8).toMillis();
```

Comment:
I mean, you also could add  this one: Configuration.pageLoadTimeout = ofSeconds(300).toMillis();
but first read about this config and maybe share info with others

Reply:
Default value for PageLoadTimeout is 30 seconds, this is enough for me. But default value for timeout (4 sec) is not enough because then i often get "Element not found" error.